app/src/main/java/base/apptrailers/mobile/globant/com/moviesapp/fragments/MovieDetailFragment.java

```diff
@@ -0,0 +1,31 @@
+package base.apptrailers.mobile.globant.com.moviesapp.fragments;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import base.apptrailers.mobile.globant.com.moviesapp.R;
+
+/**
+ * Created by raul.striglio on 07/09/16.
+ */
+public class MovieDetailFragment extends Fragment {
+
+    public String TAG = "movieDetailFragment";
+    private View rootView;
```

Comment:
you have a member attribute but you just assign it (check for AS warnings) :stuck_out_tongue: